doc/modules/computing.rst

```diff
@@ -63,27 +63,27 @@ balances relevancy and memory footprint could involve some tuning [1]_.
 Here is a list of incremental estimators for different tasks:
 
   - Classification
-      + :class:`sklearn.naive_bayes.MultinomialNB`
-      + :class:`sklearn.naive_bayes.BernoulliNB`
-      + :class:`sklearn.linear_model.Perceptron`
-      + :class:`sklearn.linear_model.SGDClassifier`
-      + :class:`sklearn.linear_model.PassiveAggressiveClassifier`
-      + :class:`sklearn.neural_network.MLPClassifier`
+      + :class:`~sklearn.naive_bayes.MultinomialNB`
```

Comment:
For these, I think including the module is useful. In other words: `naive_bayes.MultinomialNB`

Reply:
How do I do this? Would I need to set `.. currentmodule:: sklearn` at the top of the file and then not use the tilde sign?

Reply:
that's definitely one way of doing it.